src/gatling/simulations/scenarios/Solicitor_Bails.scala

```diff
@@ -0,0 +1,1383 @@
+
+      exec(http("XUI_Bails_400_005_Have_Disability")
+        .post("/data/case-types/Bail/validate?pageId=startApplicationdisabilityDetails")
+        .headers(Headers.commonHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.case-data-validate.v2+json;charset=UTF-8")
+        .header("x-xsrf-token", "${XSRFToken}")
+        .body(ElFileBody("bodies/bails/BailsHaveDisability.json"))
+        .check(substring("applicantDisability1")))
+
+    }
+    .pause(MinThinkTime, MaxThinkTime)
+
+
+    /*==================================================================================================
+    * Would the applicant be able to join the hearing by video link? - yes
+    ====================================================================================================*/
+
+    .group("XUI_Bails_410_Video_Link") {
+
+      exec(http("XUI_Bails_410_005_Video_Link")
+        .post("/data/case-types/Bail/validate?pageId=startApplicationapplicantVideoHearingDetails")
+        .headers(Headers.commonHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.case-data-validate.v2+json;charset=UTF-8")
+        .header("x-xsrf-token", "${XSRFToken}")
+        .body(ElFileBody("bodies/bails/BailsVideoLink.json"))
+        .check(substring("videoHearing1")))
+
+    }
+    .pause(MinThinkTime, MaxThinkTime)
+
+
+    /*==================================================================================================
+    * Enter the legal representative’s details
+    ====================================================================================================*/
+
+    .group("XUI_Bails_420_Legal_Reps_Details") {
+
+      exec(http("XUI_Bails_420_005_Legal_Reps_Details")
+        .post("/data/case-types/Bail/validate?pageId=startApplicationlegalRepresentativeDetails")
+        .headers(Headers.commonHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.case-data-validate.v2+json;charset=UTF-8")
+        .header("x-xsrf-token", "${XSRFToken}")
+        .body(ElFileBody("bodies/bails/BailsLegalRepsDetails.json"))
+        .check(substring("isLegalRep")))
+
+    }
+    .pause(MinThinkTime, MaxThinkTime)
+
+
+    /*==================================================================================================
+    * Check your answers
+    ====================================================================================================*/
+
+    .group("XUI_Bails_430_Check_Your_Answers") {
+
+      exec(http("XUI_Bails_430_005_Check_Your_Answers")
+        .post(BaseURL + "/data/case-types/Bail/cases?ignore-warning=false")
+        .headers(Headers.commonHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.create-case.v2+json;charset=UTF-8")
+        .header("x-xsrf-token", "${XSRFToken}")
+        .body(ElFileBody("bodies/bails/BailsCheckYourAnswers.json"))
+        .check(jsonPath("$.id").saveAs("case_Id"))
```

Comment:
For consistency with other case types, please call this caseId

Reply:
is there a quick way to do this rather than do it manually?